examples/audio/audio_stream_effects.c

```diff
@@ -14,6 +14,7 @@
 #include "raylib.h"
 
 #include <stdlib.h>         // Required for: NULL
+#include <stdbool.h>        // Required for: bool
```

Comment:
It is already included by "raylib.h" when required, no need to include it again.